packages/validator/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bubblewrap/validator",
-  "version": "1.20.0",
```

Comment:
nit: You don't necessarily have to update the version for this one (I think it hasn't changed?). The validations here are outdated too.

Reply:
That's right. I'll keep its version at 1.20.0. We can address the validator checks in a future release.